tests/view/renderer.js

```diff
@@ -1683,6 +1683,36 @@ describe( 'Renderer', () => {
 			);
 		} );
 
+		// #1451
+		it( 'should correctly render changed children even if direct parent is not marked to sync', () => {
```

Comment:
This test is a bit too hard to read. Better variable naming + a comment what's happening inside it would help.